build/lib/gradio/interface.py

```diff
@@ -42,6 +43,7 @@ def __init__(self, fn, inputs, outputs, verbose=False, examples=None,
                  title=None, description=None, thumbnail=None, 
                  server_port=None, server_name=networking.LOCALHOST_NAME,
                  allow_screenshot=True, allow_flagging=True,
+                 embedding_fn="default",
```

Comment:
since the kwarg for interpretation is interpretation="default", maybe embedding="default" for consistency, instead of embedding_fn?